jet-fuel/test/integration/url_redirect_test.rb

```diff
@@ -10,4 +10,13 @@ class UrlRedirectTest < ActionDispatch::IntegrationTest
     assert page.has_content?("Jet Fuel")
     assert page.has_field?("url[full_url]")
   end
+
+  test "As a guest, when I give a url I get back a service shortened URL" do
+    long_url = "www.thisisareallylongurl.com/1234567"
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.